project/Dependencies.scala

```diff
@@ -143,16 +141,15 @@ object Dependencies {
   }
 
   object flink {
-    lazy val version        = "1.3.2"
+    lazy val version        = "1.7.0"
     lazy val namespace      = "org.apache.flink"
-    lazy val core           = namespace % "flink-core" % version
-    lazy val streamingScala = namespace % "flink-streaming-scala_2.11" % version
+    lazy val streamingScala = namespace %% "flink-streaming-scala" % version
   }
 
   object kafka {
-    private lazy val version   = "1.0.0"
+    private lazy val version   = "2.1.1"
```

Comment:
IMHO until we'll open a new repository for `nsdb-connector`, we might support latest minor for **major 1**, i.e. `kafka-1.1.1`.

Reply:
Agreed. I will update this version